madgiraffestudios/src/components/About/About.js

```diff
@@ -0,0 +1,100 @@
+                </div>
+                <div>
+
+                </div>
+
+                <div className="row">
+                    <div className="col-md-6 mt-5 p-3" id="second">
+                        <h5 className="font-weight-bold primary-color">Graphic Design</h5><p className="w-75 text-muted">Your improvement as a business is my priority, what a business displays is extremely important to how they’re portrayed. Using custom logos, and having a strong Social media presence graphically will truly boost Company Branding and popularity.</p>
+                    </div>
+                    <div className="col-md-6" id="first">
+                        <img src={graphicdesign} alt="performance" className="img-fluid rounded float-right" />
+                    </div>
+                </div>
+
+
+                <div className="row">
+                    <div className="col-md-6" id="first">
+                        <img src={drawing} alt="performance" className="img-fluid rounded float-left" />
+                    </div>
+                    <div className="col-md-6 mt-5 p-3" id="second">
+                        <h5 className="font-weight-bold primary-color">Illustrations</h5>
+                        <p className="w-75 text-muted">Now imagine having your favorite photo of yourself, or a loved one reimagined through in an Illustration.Choose from a hand drawing,painting or digital art and let me bring your imagination to life.                                                                                            Get started today!
+                           </p>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+
+
+        <div class="footer_bg pt-5 pb-3 bg-light text-center">
+            <div class="container">
+                <div class="row">
+                    <div class=" col-md-4 text-center">
+                        <a href={HOME}> <img src={giraffeicon} height="50px" alt="logo" id="footer-logo" /></a>
+                    </div>
+                    <div class=" col-md-4 text-center">
+                        <div class="row">
+                            <div className="col-md-2">
+
+                                <p><SocialIcon url="https://www.instagram.com/madgiraffestudios/" bgColor="#8d18b4" />
+                                </p>
+                            </div>
+                            <div className="col-md-2">
+                                <p><SocialIcon url="https://twitter.com/MadGiraffeStud1" bgColor="#8d18b4" />
+                                </p>
+                            </div>
+                            <div className="col-md-2">
+                                <p>
+                                    <SocialIcon url="https://www.linkedin.com/in/mackenzie-heslin-scott-70ab4a116/" bgColor="#8d18b4" />
```

Comment:
- [ ] Inline hexcode isn't recommended - should be set in sass for reusability, please address

Reply:
will be addressed in another issue